src/style.css

```
@@ -0,0 +1,35 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position-x: 50%;
+  background-position-y: center;
```

You don't need 2 different rules 
```suggestion
  background-position: center;
```